src/components/Order/Stepper.styles.js

```diff
@@ -0,0 +1,26 @@
+export const containerStyle = {
+  width: "100%",
+};
+
+export const stepContentBox = {
+  my: 4,
```

Comment:
About the comment in the component - this is where I would change the names of your object (add Style at the end)